sdk/samples/iot/iot_sample_common.c

```diff
@@ -41,32 +41,22 @@ do {                                          \
       exit(1);                                \
     }                                         \
 } while (0)
-*/
-
-static inline void precondition_not_null(void const* arg)
-{
-  if (arg == NULL)
-  {
-    LOG_ERROR("Pointer is NULL.");
-    exit(1);
-  }
-}
 
 //
 // MQTT endpoints
 //
 //#define USE_WEB_SOCKET // Comment to use MQTT without WebSockets.
 #ifdef USE_WEB_SOCKET
-az_span const IOT_SAMPLE_MQTT_URL_PREFIX = AZ_SPAN_LITERAL_FROM_STR("wss://");
+static az_span const mqtt_url_prefix = AZ_SPAN_LITERAL_FROM_STR("wss://");

```

Comment:
Why did we rename these back?

Reply:
because theses are static globals and the globals the C guidelines speak of appear to be true globals.  However, it leaves one to assume that `extern` must have been used in some other file not shown.   The C guidelines do, however, speak to static functions as opposed to public or private/internal API functions, so I am following that standard (static being the common denominator) that there does not need to be a prefix.  

I have added this question on static globals to the list of C guideline topics to address. https://gist.github.com/momuno/70fabd6609b6459cc3599f1ba037af42